### DISCUSSION TITLE
Two questions before install

### DISCUSSION THREAD

**Discussion by Str1atum** [marcone/teslausb#868] [Q&A]
Hi all,

I am shortly before reinstalling teslausb on new hardware (Pi Zero 2 W is ready) after my old Pi was destroyed in a fire together with the Tesla.

So two questions before installing:
1. Should I use [teslausb-20231012-bullseye] or [teslausb-20211106.2] image?
2. Can I test the sync process without a Tesla, so e.g. just mounting the USB drive like the Tesla would and put some files there? Or do they need to be in specific folders to work? I do not intend music syncing or other file sync than Sentry cam.

Thank you for your help,
Str1atum

**Comment by marcone** [comment]
I recommend using the teslausb-20211106.2 image. Files need to be in specific folders, and be named according to the naming scheme the car uses.

**Reply by f4llenz** [reply]
I installed the bullseye version because I assumed it'd be better as it's more recent. It works but sometimes my Model Y won't recognize that there's something plugged in the USB port, my Pi Zero 2W keeps blinking when this happens.

Should I reflash it to the 2021 release?

**Reply by marcone** [reply]
> It works but sometimes my Model Y won't recognize that there's something plugged in the USB port

If your car is not on firmware 2024.26 yet, that's probably why. Firmware prior to that had USB problems for months.

> my Pi Zero 2W keeps blinking when this happens.

It's normal for it to blink.
 
> Should I reflash it to the 2021 release?

Worth a try probably.
